src/lib/components/steppers/Steppers.component.js

```diff
@@ -43,6 +47,11 @@ const Circle = styled.div`
         background-color: ${brandingTheme.primary};
         color: ${defaultTheme.white};
       `;
+    } else if (props.error) {
```

Comment:
A step could be:
active => primary color
active and inProgress => primary color with loader
active and error => danger color
completed => green with check icon
default => `gray` color (next steps)